app/app.go

```diff
@@ -478,17 +484,36 @@ func NewStrideApp(
 	)
 	ratelimitModule := ratelimitmodule.NewAppModule(appCodec, app.RatelimitKeeper)
 
+	// Initialize the packet forward middleware Keeper
+	// It's important to note that the PFM Keeper must be initialized before the Transfer Keeper
+	app.PacketForwardKeeper = packetforwardkeeper.NewKeeper(
+		appCodec,
+		keys[packetforwardtypes.StoreKey],
+		// app.GetSubspace(packetforwardtypes.ModuleName), TODO: why is this in the integration docs but not in code?
+		app.TransferKeeper, // will be zero-value here, reference is set later on with SetTransferKeeper.
```

Comment:
should this be nil?

Reply:
app.GetSubspace or app.TransferKeeper?

If it's `app.TransferKeeper`, I didn't change anything (but my guess is it's typed so we can't pass `nil`)

Reply:
I think it's supposed to be `nil` per the integration [docs](https://github.com/cosmos/ibc-apps/pull/147/files#diff-6264bd95a51d36427689e0470a4c3300334d62154d1e89baed20b0a00a9323bfR59)

It's set after the keeper is created [[here](https://github.com/cosmos/ibc-apps/pull/147/files#diff-6264bd95a51d36427689e0470a4c3300334d62154d1e89baed20b0a00a9323bfR80)]

Reply:
Updated